### ISSUE TITLE
Docs: Gameclient assets generation/import documentation request

### ISSUE THREAD

**Issue by SakuraIsayeki** [WoWs-Builder-Team/minimap_renderer#10]
**Is your feature request related to a problem? Please describe.**
As a downstream/fork developer, I would like to be able to provide my own WOWS gameclient version assets, so as to streamline my own releases. However, I have no information as to how to achieve that.

**Describe the solution you'd like**
Please provide some documentation on how you import game client assets to the project, and generate these commits (such as https://github.com/WoWs-Builder-Team/minimap_renderer/commit/9f1add30b3fc91bffa50bb1f03fc93313c461e14).

**Comment by chemputer** [comment]
I agree, currently, we've just got to wait for them to update for the new version and it's sort of dark magic to the rest of us. A script that does it automatically provided with the game client version and replays_unpack that supports the version would be even better, but, hey, I'll settle for some documentation. I'm sure many of us would be glad to pitch in on the work if they can make reasonably useful documentation on it.

I believe for some files they use the scripts [here](https://github.com/WoWs-Builder-Team/minimap_renderer/tree/master/renderer_data/generators) with an up-to-date `GameParams.data` in `resources/`. many others are just extracted from the game files using WoWS Unpacker, and some are from a script run on those files, but many are not remotely obvious how to obtain/modify, and that's the problem.

Personally, I'd like to know what modifications to [replays_unpack](https://github.com/monstrofil/replays_unpack) are needed to make it functional for this.

**Comment by HenryQuan** [comment]
@chemputer If they are updating using some scripts, I have a tiny machine to automate some tasks every day, so potentially, I can include this project and submit a PR the same day the client updates.
